src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,31 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+interface Props {
+  todo: Todo;
+  userId: number;
+  onClose: React.Dispatch<React.SetStateAction<Todo | null>>;
+}
+
+export const TodoModal: React.FC<Props> = ({ todo, onClose, userId }) => {
+  const [user, setUser] = useState<User | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
sometimes you can see todoModal for a second before loader appears so it would be better for isLoading to have a default value of true